### PR TITLE
change license from GPL3 to GPL2

### LINKED ISSUE #5: Wrong license
Hello Mittwald-Team,

as TYPO3 need left over license, you have to add GPL V2 instead of V3.

Stefan

### PR DESCRIPTION
**Pull request by dermodmaster**
Resolves #5

### CODE REVIEW

File: ext_emconf.php [hunk 1]
```diff
@@ -12,11 +12,7 @@
   'author_email' => 'support@mittwald.de',
   'author_company' => 'Mittwald CM Service GmbH',
   'state' => 'beta',
-<<<<<<< HEAD
-  'version' => '1.1.0',
-=======
-  'version' => '1.0.0',
->>>>>>> parent of 812eb1f... v1.10
+  'version' => '1.1.1',
```
Comment (martin-helmich) [inline ext_emconf.php:15]: Please raise the version number separately.

File: Classes/Widgets/AbstractDoughnutChartWidget.php [hunk 2]
```diff
@@ -12,11 +12,10 @@
  *  This script is part of the TYPO3 project. The TYPO3 project is
  *  free software; you can redistribute it and/or modify
  *  it under the terms of the GNU General Public License as published by
- *  the Free Software Foundation; either version 2 of the License, or
- *  (at your option) any later version.
+ *  the Free Software Foundation; either any later version.
```
Comment (martin-helmich) [inline Classes/Widgets/AbstractDoughnutChartWidget.php:15]: Is this search&replace gone wrong? The official recommended text from the license v2 reads:

```
This program is free software; you can redistribute it and/or
modify it under the terms of the GNU General Public License
as published by the Free Software Foundation; either version 2
of the License, or (at your option) any later version.
```

Comment (martin-helmich) [reply Classes/Widgets/AbstractDoughnutChartWidget.php:15]: Also, this was definitely not the original recommended license notice -- and this software is (arguably) not "part of the TYPO3 project". Did we plagiarize this license notice from somewhere else?

File: ext_emconf.php (hunk 1 above)
Comment (martin-helmich) [inline ext_emconf.php:15]: Also, there should also be a reference to the library in the `composer.json` file. Please change that also (or add, if not already present).

**Review by martin-helmich — CHANGES_REQUESTED** [review]

**Comment by derhansen** [comment]
Closed with bb1a97c13b0721c93a28075791c3bf44711df0af

**Pull request closed without merging**